### PR TITLE
Updating cache contributors list

### PR DESCRIPTION
**Pull request by dragoonis**

### CODE REVIEW

**Comment by dragoonis** [comment]
@Crell ping - can you merge?

File: proposed/cache-meta.md [hunk 1]
```diff
@@ -279,16 +279,19 @@ do so.
...
+### 5.3 Sponsors
 
-* Beau Simensen, Sculpin (Coordinator)
+* Beau Simensen, Sculpin
 * Pádraic Brady, Zend Framework
 
-### 5.3 Contributors
+### 5.4 Contributors
 
 * Paul Dragoonis
```
Comment (Crell) [inline proposed/cache-meta.md]: Per the blyaw, Editor, Coordinator, and Sponsor should not be listed in the Contributors list.

File: proposed/cache-meta.md [hunk 2]
```diff
@@ -279,16 +279,19 @@ do so.
 5. People
 ---------
 
-### 5.1 Editor
+### 5.1 Coordinator
+* Paul Dragoonis
```
Comment (Crell) [inline proposed/cache-meta.md]: No other spec lists the Coordinator separately.  It just lists the Sponsors and Coordinator is marked, as below.

File: proposed/cache-meta.md (hunk 1 above)
Comment (dragoonis) [inline proposed/cache-meta.md]: @Crell okay, amended. refresh.

File: proposed/cache-meta.md [hunk 3]
```diff
@@ -279,18 +279,20 @@ do so.
...
+### 5.2 Editor
 
 * Larry Garfield
 
-### 5.2 Sponsors
+### 5.3 Sponsors
 
-* Beau Simensen, Sculpin (Coordinator)
+* Beau Simensen, Sculpin
 * Pádraic Brady, Zend Framework
```
Comment (Crell) [inline proposed/cache-meta.md]: This should list Paul as a Sponsor with (Coordinator) after him, just replacing Beau directly.  (I'd be fine with listing Beau as an additional contributor if you wanted, or not. Either way.)

Comment (dragoonis) [inline proposed/cache-meta.md]: @Crell so I'd be cool with me under Contributor section with (coordinator). I've never been a sponsor.

**Comment by Crell** [comment]
Paul, I'm in IRC for a bit if you want to talk there.

**Pull request merged**